### DISCUSSION TITLE
请问docker安装后如何更新到新版本

### DISCUSSION THREAD

**Discussion by sjqboss** [AlistGo/alist#1535] [Q&A]
在说明里只有docker下的全新安装命令，那个一键脚本的更新命令应该不适用docker吧？

**Comment by sunbingrun** [comment]
那肯定的， 手动更新镜像吧，或者 试试 containrrr/watchtower 镜像

**Comment by GitCourser** [comment]
3个方法
1.手动停止旧容器,下载新镜像后按之前的配置重新启动容器,然后删除旧的镜像
2.用watchtower镜像可以建立一个自动检查更新容器的容器
3.用Portainer可以对某个容器重新创建,创建时选择重新拉取最新镜像
